libpod/container_log.go

```diff
@@ -81,5 +83,33 @@ func (c *Container) readFromLogFile(options *logs.LogOptions, logChannel chan *l
 		}
 		options.WaitGroup.Done()
 	}()
+	// Check if container is still running or paused
+	if options.Follow {
+		go func() {
+			for {
+				state, err := c.State()
+				if err != nil {
+					time.Sleep(250 * time.Millisecond)
```

Comment:
Rather then sprinkle magic numbers here this should be
time.Sleep(watch.POLL_DURATION)
Then future developers would have an idea where this came from.

Reply:
@rhatdan Thanks for the suggestion; definitely a better way to do it. I was able to also refactor it down to a single sleep statement with no change to the logic of the loop. This has passed all checks and should be good to go now.